airflow/models/dagbag.py

```diff
@@ -390,7 +406,17 @@ def _process_modules(self, filepath, mods, file_last_changed_on_disk):
     def bag_dag(self, dag, root_dag):
         """
         Adds the DAG into the bag, recurses into sub dags.
-        Throws AirflowDagCycleException if a cycle is detected in this dag or its subdags
+
+        :throws: AirflowDagCycleException if a cycle is detected in this dag or its subdags.
+        :throws: AirflowDagDuplicatedIdException if this dag or its subdags already exists in the bag.
```

Comment:
Shouldn't this be `:raises:` ?

```suggestion
        :raises: AirflowDagCycleException if a cycle is detected in this dag or its subdags.
        :raises: AirflowDagDuplicatedIdException if this dag or its subdags already exists in the bag.
```

Reply:
You're right, I don't know why I wrote this.